tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "target": "ES2015",       /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
+    "module": "commonjs",     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    "lib": ["es2016"],        /* Specify library files to be included in the compilation. */
+    "allowJs": true,          /* Allow javascript files to be compiled. */
+    "checkJs": true,          /* Report errors in .js files. */
+    "noEmit": true,           /* Do not emit outputs. */
+
+    /* Strict Type-Checking Options */
+    // "strict": true,                           /* Enable all strict type-checking options. */
+    // "noImplicitAny": false,                 /* Raise error on expressions and declarations with an implied 'any' type. */
+    // "strictNullChecks": true,              /* Enable strict null checks. */
```

Comment:
The flag `strictNullChecks` is left unenabled because there is about 20 places where I need to either use `@ts-ignore` or write very awkward codes.